fs/copy_unix_test.go

```diff
@@ -0,0 +1,87 @@
+// +build !windows
```

Comment:
`sysx` xattr support is `// +build linux darwin`.  When I attempted to add `freebsd` locally, I got errors like this:

```
copy_unix_test.go:66: failed to copy xattrs: failed to get xattr "\x06test-2\x06test-x\x06test-1" on /tmp/test-copy-src-with-xattr-113390834: attribute not found
```

I think since xattrs are not supported by continuity on FreeBSD, it's better to match the build tags used by the xattr code.  I do realize that I specifically asked you to include FreeBSD earlier; I apologize for asking you to do that without actually knowing that it would work.

```suggestion
// +build linux darwin
```